app/Resources/assets/js/jquery.instantSearch.js

```diff
@@ -0,0 +1,79 @@
+// jQuery plugin for an instant searching
+(function($) {
+    $.fn.instantSearch = function(config) {
+        return this.each(function() {
+            initInstantSearch(this, $.extend(true, defaultConfig, config || {}));
+        });
+    };
+
+    var defaultConfig = {
+        minQueryLength: 2,
+        maxPreviewItems: 10,
+        previewDelay: 500,
+        noItemsFoundMessage: 'No items found'
+    };
+
+    var throttle = (function(){
+        var timer = 0;
+        return function(callback, ms){
+            clearTimeout (timer);
+            timer = setTimeout(callback, ms);
+        };
+    })();
+
+    var initInstantSearch = function(el, config) {
+        var $input = $(el);
+        var $form = $input.parents('form').first();
+        var $preview = $('<ul class="search-preview list-group"></ul>').appendTo($form);
+
+        var setPreviewItems = function(items) {
+            $preview.empty();
+
+            $.each(items, function(index, item) {
+                if (index > config.maxPreviewItems) {
+                    return;
+                }
+
+                addItemToPreview(item)
```

Comment:
OK. Then I ask for this change because of our own code consistency. Either add this semicolon ... or remove all the other semicolons. Thanks!